configs/lastNotifiedTransactions.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This file should be named in underscore convention to be consistent with other json files.